cmd/submit_test.go

```diff
@@ -434,3 +483,19 @@ func writeFakeSolution(t *testing.T, dir, trackID, exerciseSlug string) {
 	err := solution.Write(dir)
 	assert.NoError(t, err)
 }
+
+func writeFakeLegacySolution(t *testing.T, dir, trackID, exerciseSlug string) {
```

Comment:
This is only used in one place, right? If so I'd rather inline it into the test.